docs/making-release-notes.md

```diff
@@ -0,0 +1,38 @@
+## Making release notes
+This documents the process for making release notes for a release.
+
+### 1) Note the PR number of the previous release
+Find the PR that was merged with the previous release.  Remember this number
```

Comment:
TODO is fine. Just wondering if we needed to add a step to write this down to save the next release engineer a ton of time or if there's an easy way to regenerate it.